### PR TITLE
Admin Check exam

### PR DESCRIPTION
**Pull request by NgengChhengkim**

### CODE REVIEW

File: app/helpers/admin/exams_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module Admin::ExamsHelper
```
Comment (RathanakSreang) [inline app/helpers/admin/exams_helper.rb]: remove this file

**Comment by RathanakSreang** [comment]
Ok

**Comment by HorNorin** [comment]
OK

File: app/views/admin/exams/_find_status.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="find-exam-status">
+  <%= form_for :exam, html: {id: "frm-status"}, url: admin_exams_path, method: "get", remote: true do |f| %>
```
Comment (chuanhtuan) [inline app/views/admin/exams/_find_status.html.erb]: <%= form_for [:admin, :exam], html: {id: "frm-status"}, method: "get", remote: true do |f| %>

**Pull request merged**